.ci/packaging.groovy

```diff
@@ -127,11 +143,19 @@ pipeline {
                 sh(label: 'create dependencies file', script: 'make release-manager-dependencies-snapshot')
               }
               dockerLogin(secret: env.DOCKER_SECRET, registry: env.DOCKER_REGISTRY)
-              script {
-                getVaultSecret.readSecretWrapper {
-                  sh(label: 'release-manager.sh', script: '.ci/scripts/release-manager.sh')
-                }
-              }
+              releaseManager(project: 'fleet-server',
+                             version: env.VERSION,
+                             type: 'snapshot',
+                             artifactsFolder: 'build/distributions',
+                             outputFile: env.DRA_OUTPUT)
+            }
+          }
+          post {
+            failure {
+              notifyStatus(analyse: true,
+                           file: "${BASE_DIR}/${env.DRA_OUTPUT}",
+                           subject: "[${env.REPO}@${env.BRANCH_NAME}] DRA failed.",
+                           body: 'Contact the Release Platform team [#platform-release].')
```

Comment:
So if something fails, the only thing to do is to contact the release platform team right?

Reply:
IIUC, the `DRA` component is a black box, so if the packages have been generated correctly, and the `DRA` stage fails then I'd say that's the way to go.

If the package generation fails earlier, then that's something to be investigated, though that's a unlikely scenario